### DISCUSSION TITLE
LATERAL Not Implemented

### DISCUSSION THREAD

**Discussion by wagnerc4** [duckdb/duckdb#5733] [Ideas]
When I try to use LATERAL with duckdb I get this:
NotImplementedException: Not implemented Error: LATERAL not implemented

I cant switch from postgresql if LATERAL is not implemented.

**Comment by Alex-Monahan** [comment]
Could you try on the latest master? We have partial support for Lateral already merged in, just not released!

https://github.com/duckdb/duckdb/pull/5485
https://github.com/duckdb/duckdb/pull/5393

**Reply by wagnerc4** [reply]
Hello! thanks for fast reply.
I try: pip install duckdb --pre --upgrade     but install the same version 0.6.1
Also try: pip install git+https://github.com/duckdb/duckdb.git@master     but get error: ERROR: git+https://github.com/duckdb/duckdb.git@master does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.

I will have to use make???

**Reply by Alex-Monahan** [reply]
Would you mind retrying the pip command? We have some more recent versions uploaded to pypi as of a few hours ago.

https://pypi.org/project/duckdb/#history

**Reply by wagnerc4** [reply]
I retry: pip install duckdb --pre --upgrade
and now installed: duckdb-0.6.2.dev702
Then I make a simple query with LATERAL and works perfect!
Thank you!
